server/routes/coordinator.js

```diff
@@ -80,11 +79,14 @@ module.exports = function Index({ reportService, involvedStaffService, reviewSer
         return res.redirect(`/coordinator/report/${reportId}/confirm-delete`)
       }
 
+      const report = await reviewService.getReport(reportId)
+      const referringPage = report.status === 'SUBMITTED' ? '/not-completed-incidents' : '/completed-incidents'
```

Comment:
We have a constant you can refer to here: `ReportStatus.SUBMITTED.value`

`import { ReportStatus } from '../config/types'`

Reply:
Constant used. 
Could not use the import statement because it was giving a babel error. 
Used _const { ReportStatus } = require('../config/types')_ instead